docs/specs/manifest.yml

```diff
@@ -55,7 +55,7 @@ outputs:
     {
       "files": [
         { "url": "/c", "!hash": null },
-        { "url": "/a", "hash": "cc98fe5ff90ff67265c7d6f3f7ed70298d2f19a7" },
+        { "url": "/a", "hash": "d9f0ca10e7db6c5a0a5e0c3279d38576aea69869" },
```

Comment:
this is due to `raw_title=""` --> `!raw_title`. docs-ui does not depend on raw_title having a value, so the change is safe.